suite-common/wallet-core/src/accounts/accountsReducer.ts

```diff
@@ -386,7 +390,7 @@ export const selectIsDeviceAccountless = (state: AccountsRootState & DeviceRootS
     pipe(selectVisibleDeviceAccounts(state), A.isEmpty);
 
 // Selected device has no accounts and no active discovery. It can be empty portfolio device.
-export const selectIsEmptyDevice = (
+export const selectIsDiscoveredAccountlessDevice = (
```

Comment:
nitpick, feel free to ignore
```suggestion
export const selectIsDiscoveredDeviceAccountless = (
```

Reply:
e6791b5ea8